lib/comfortable_mexican_sofa/extensions/is_taggable.rb

```diff
@@ -8,7 +8,7 @@ module ClassMethods
         # Comfy already define #tags and #tags= methods in some models, so let's call :keywords to the Tag associated
         # instances of a model to avoid naming collisions!!!
         # Creates the appropriate AR associations, scopes and instance methods.
-        def is_taggable
+        def is_taggable # rubocop:disable Style/PredicateName
```

Comment:
Just one thing, after talk with @neoeno I will remove this method, because is just including a module. We can include on the Page model now, in  order to be more explicit without any magic stuff.